reindexer/terraform/terraform.tf

```diff
@@ -8,3 +8,23 @@ terraform {
     region         = "eu-west-1"
   }
 }
+
+data "terraform_remote_state" "shared_infra" {
+  backend = "s3"
+
+  config {
+    bucket = "wellcomecollection-platform-infra"
+    key    = "terraform/shared_infra.tfstate"
+    region = "eu-west-1"
+  }
+}
+
+data "terraform_remote_state" "catalogue_pipeline_data" {
```

Comment:
nit: the remote state file in S3 doesn’t match the name, but the new name for this is "infra_critical".